internal/tiltfile/extension/extension.go

```diff
@@ -46,6 +47,12 @@ func (e *Extension) LocalPath(t *starlark.Thread, arg string) (string, error) {
 		return "", nil
 	}
 
+	loadIsHappeningInTopLevel := t.CallStackDepth() == 1
```

Comment:
Is this a uhh ... valid check?

Reply:
I thought we agreed to ban it for now, though if we want to revisit that decision I'm down.